### DISCUSSION TITLE
Is there a way to see the fonts available in the web app?

### DISCUSSION THREAD

**Discussion by cds-internationalist** [typst/typst#1225] [Q&A] [answered]
As per title, is there a way to see the fonts available in the web app?

**Comment by LaurenzV** [comment, marked as answer]
<img width="223" alt="image" src="https://github.com/typst/typst/assets/47084093/646f67cf-664e-4421-93fc-34d93db629d3">
If you press on the "Ag" in the top bar you see a list of available fonts.

**Reply by cds-internationalist** [reply]
Thank you! I knew I was missing something!
